grassroot-webapp/src/main/java/za/org/grassroot/webapp/controller/rest/group/GroupModifyController.java

```diff
@@ -68,15 +71,19 @@
     private final AccountBroker accountBroker;
     private final AccountFeaturesBroker accountFeaturesBroker;
     private final GroupStatsBroker groupStatsBroker;
+    private final GeoLocationBroker geoLocationBroker;
+
 
     public GroupModifyController(JwtService jwtService, UserManagementService userManagementService, GroupFetchBroker groupFetchBroker,
-                                 GroupImageBroker groupImageBroker, AccountBroker accountBroker, AccountFeaturesBroker accountFeaturesBroker, GroupStatsBroker groupStatsBroker) {
+                                 GroupImageBroker groupImageBroker, AccountBroker accountBroker, AccountFeaturesBroker accountFeaturesBroker,
+                                 GroupStatsBroker groupStatsBroker,GeoLocationBroker geoLocationBroker) {
         super(jwtService, userManagementService);
         this.groupFetchBroker = groupFetchBroker;
         this.groupImageBroker = groupImageBroker;
         this.accountBroker = accountBroker;
         this.accountFeaturesBroker = accountFeaturesBroker;
         this.groupStatsBroker = groupStatsBroker;
+        this.geoLocationBroker = geoLocationBroker;
```

Comment:
Are you actually using that broker in this controller? Doesn't look like it. So remove.